docs/reference/aggregations/bucket/daterange-aggregation.asciidoc

```diff
@@ -61,6 +61,43 @@ Response:
 --------------------------------------------------
 // TESTRESPONSE[s/\.\.\./"took": $body.took,"timed_out": false,"_shards": $body._shards,"hits": $body.hits,/]
 
+==== Missing Values
+
+The `missing` parameter defines how documents that are missing a value should be treated.
+By default they will be ignored but it is also possible to treat them as if they had a value.
+This is done by adding a set of fieldname : value mappings to specify default values per field.
```

Comment:
This sentence makes me a bit confused due to the fact that date-range aggregations only work on a single field, so you would only have one mapping for the missing value (rather than "a set of")?